### ISSUE TITLE
Tab order and hot keys

### ISSUE THREAD

**Issue by juliainkster** [bcgov/name-examination#1093]
As a NE I want to be able to tab easily through the most used elements so that I don’t have to use my mouse so that I can process NRs quicker. 

Most common tasks should be keyboard navigable.

**Comment by LJTrent** [comment]
This is addressed through the code with us phases.
